### PR TITLE
feat(cli): support multiple locales in extract command

### PR DESCRIPTION
**Pull request by dan-dr**
# Description
Adds multiple locales support for `--locale` option, similar to the experimental extract implementation.

## Types of changes

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation update
- [ ] Examples update

## Checklist

- [x] I have read the [CONTRIBUTING](https://github.com/lingui/js-lingui/blob/main/CONTRIBUTING.md) and [CODE_OF_CONDUCT](https://github.com/lingui/js-lingui/blob/main/CODE_OF_CONDUCT.md) docs
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added the necessary documentation (if appropriate)

### CODE REVIEW

**Review by andrii-bodnar — APPROVED** [review]
@dan-dr thank you!

File: packages/cli/src/lingui-extract.ts [hunk 1]
```diff
@@ -103,15 +103,21 @@ type CliOptions = {
...
 
 if (require.main === module) {
   program
     .option("--config <path>", "Path to the config file")
-    .option("--locale <locale>", "Only extract the specified locale")
+    .option(
+      "--locale <locale, [...]>",
+      "Only extract the specified locales",
+      (value) => {
+        return value.split(",").filter(Boolean)
```
Comment (vonovak) [inline packages/cli/src/lingui-extract.ts:118]: shouldn't the values be trimmed? https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/trim

**Comment by dan-dr** [comment]
What needs to happen to merge? :)

**Comment by vonovak** [comment]
Hi! Strictly speaking, you first want to do the trimming and then the  filtering 🙈🙂

**Comment by dan-dr** [comment]
> Hi! Strictly speaking, you first want to do the trimming and then the filtering 🙈🙂

yikes! 😨

**Comment by dan-dr** [comment]
done

**Comment by andrii-bodnar** [comment]
@dan-dr It has been merged into the `next` branch and will be released in Lingui v4.

**Comment by dan-dr** [comment]
got it. why though? could be useful right away

**Pull request merged**